CMakeLists.txt

```diff
@@ -1,10 +1,23 @@
 project(doppelkopf)
 cmake_minimum_required(VERSION 3.12)
 
+set(gtest_force_shared_crt ON)
 
-#add_subdirectory(libs/googletest)
 add_subdirectory(src)
-
 add_executable(doppelkopf ${SOURCE} ${CORE})
```

Comment:
I think it also would be better to have all exectuables at one place, f.e. a build folder, that we can put in the .gitignore.

Reply:
https://cmake.org/pipermail/cmake/2006-October/011711.html this should do the trick

Reply:
I propose add a dedicated build directory, and then adding `set( CMAKE_RUNTIME_OUTPUT_DIRECTORYbuild)` 
For the tests I am not sure if the should be also in this extra build folder or if we want to have a seperation

Reply:
this only outputs the exe into the /bin folder, but not all the cmake files like `CMakeCache.txt` and others. Therefore it is not a solution

Reply:
I see. Then let's just document it in the readme and leave it as is.